modules/aks/kubernetes.tf

```diff
@@ -1,12 +0,0 @@
-provider "kubernetes" {
```

Comment:
Why did you removed this block?

Reply:
I removed it because that provider is not used anywhere, but maybe I'm missing some context about why it was there in the first place.

Reply:
Ye.... i put it there to "later" extend installer feature (example: adding some auto-deployable components like fury-kubernetes-opa)

But, as you saw, it is currently unused.... so i don't know what to suggest.... let it there or remove it....

Reply:
Fair enough, I dropped that commit.